dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -138,6 +138,7 @@ jersey-server/3.0.12//jersey-server-3.0.12.jar
 jettison/1.5.4//jettison-1.5.4.jar
 jetty-util-ajax/11.0.21//jetty-util-ajax-11.0.21.jar
 jetty-util/11.0.21//jetty-util-11.0.21.jar
+jjwt-api/0.12.6//jjwt-api-0.12.6.jar
```

Comment:
Do we need to update our NOTICE-binary?

Reply:
Thanks. Sure, @yaooqinn ! It's Apache License. Let me add this item.

Reply:
Thank you for the update! @dongjoon-hyun